packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -232,15 +247,22 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
     }
   }
 
+  @autobind
   private _onSelectNextMonth() {
     this.props.onNavigateDate(addMonths(this.props.navigatedDate, 1), false);
   }
 
+  @autobind
   private _onSelectPrevMonth() {
     this.props.onNavigateDate(addMonths(this.props.navigatedDate, -1), false);
   }
 
   @autobind
+  private _onHeaderClick(focus: boolean) {
+    this.props.onHeaderClick ? this.props.onHeaderClick(focus) : null;
```

Comment:
let { onHeaderClick } = this.props;
if (onHeaderClick) {
    onHeaderClick(focus);
}